src/utility.cpp

```diff
@@ -515,106 +558,128 @@ BOOL PatchImport (HMODULE importmodule, moduleentry_t *patchModule)
 	LPSTR pszBuffer = cwBuffer;
 	DWORD dwMaxChars = _countof(cwBuffer);
 	DWORD dwLength = ::GetModuleFileNameA(importmodule, pszBuffer, dwMaxChars);
-#endif
+#endif
+
+    // have a stack local array of the addresses, don;t want to use malloc for this

```

Comment:
>// have  [](start = 4, length = 8)

Probably good to have a comment here why we cache the real addresses/VirtualProtect is expensive (for "future developers")

Reply:
Added

---
In reply to: [591985755](https://github.com/Azure/vld/pull/8#discussion_r591985755) [](ancestors = 591985755)